neo/rawio/neuralynxrawio/NcsBlocks.py

```diff
@@ -0,0 +1,416 @@
+import math
+
+
+class NcsBlocks:
+    """
+    Contains information regarding the contiguous blocks of records in an Ncs file.
+    Methods of NcsBlocksFactory perform parsing of this information from an Ncs file and
+    produce these where the blocks are discontiguous in time and in temporal order.
+
+    TODO: This class will likely need __eq__, __ne__, and __hash__ to be useful in
+    more sophisticated segment construction algorithms.
+
+    """
+    def __init__(self):
+        self.blocks = []
+        self.sampFreqUsed = 0  # actual sampling frequency of samples
+        self.microsPerSampUsed = 0  # microseconds per sample
+
+
+class NcsBlock:
```

Comment:
Agreed. Had raised this issue on the Google group. Have changed to 'ncs_section' and so on because both block and segment are already used for core concepts in Neo.